resources/views/posts/create.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts/app')
+
+@section('title','Create Post')
+
+@section('content')
+    <div class="row justify-content-center">
+        <div class="col-7">
+            <h1>Create Post</h1>
+        </div>
+    </div>
+    
+    <div class="row justify-content-center">
+        <div class="col-7">
+            <hr>
+            <form action="/posts" method="POST">
```

Comment:
Same here. I believe you can use URL::route() for post urls as well.

Reply:
implemented!